prow/config.yaml

```diff
@@ -1792,9 +1843,55 @@ presubmits:
     labels:
       preset-service-account: true
       preset-k8s-ssh: true
+      preset-bazel-scratch-dir: true
     spec:
       containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20180307-ce8e19913-master
+        args:
+        - --root=/go/src
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=k8s.io/release"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        env:
+        - name: BAZEL_REMOTE_CACHING_ENABLED
```

Comment:
is this a typo? should `CACHING` be `CACHE` instead?

Reply:
yeah these were wrong... I should probably consider moving these to a preset in the future as well, we can actually validate that those exist.

Reply:
fixed